Dockerfile

```diff
@@ -1,12 +1,18 @@
 FROM python:3.9-slim
-RUN pip install --no-cache-dir pipenv
+
+RUN useradd -u 65532 -d /app -m appuser
+USER 65532
+
 WORKDIR /app
+
 COPY Pipfile* /app/
-RUN cd /app && \
-    pipenv install --system --deploy
+COPY *.py LICENSE /app/
```

Comment:
I think we can combine these copies

Reply:
I considered it at first... but should we really do it in the same PR?
Remember, this one is for running as non-root.
I'm ok with changing it here though.

Reply:
I think combining these are ok. :)